render/query.go

```diff
@@ -218,11 +218,24 @@ func (r *Reply) getDataAggregated(ctx context.Context, cfg *config.Config, tf Ti
 	data = EmptyData
 
 	metricList := targets.AM.Series(targets.isReverse)
```

Comment:
This is used in `queryCarbonlink(ctx, cfg, metricList)` and should be always in direct order.

```suggestion
	metricList := targets.AM.Series(false)
```

See following suggestion

Reply:
Why ? Now it's used for Path list in clickhouse query. So it must to coexistense with table type.
At master for carbonlink used metricList (which can be reverse). So, set to

`carbonlinkResponseRead := queryCarbonlink(ctx, cfg, metricListUnreverse)`